.github/actions/parse-next-release-from-chart/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Parse next release from chart
+description: >
+  Parses the next main release version based on the content of Chart.yaml.
+  The suffix `-SNAPSHOT` is removed.
+inputs:
+  chart-dir:
+    description: "The relative path to the directory holding Chart.yaml"
+    required: true
+  chart-repository-dir:
```

Comment:
Good catch! Actually, I should be able to keep only `chart-repository-dir` and use `yq e '.version' **/Chart.yaml` in place of `yq e '.version' ${{ inputs.chart-dir }}/Chart.yaml`. I'll update it.

Reply:
In the end, I think it's better to keep it like that. The alternatives would be either using `*/**/Chart.yaml` to match the file, but it less specific. Or add some logic to find one of the information based on the other one, but I'd prefer to avoid that.